### ISSUE TITLE
Bump uuid dependency

### ISSUE THREAD

**Issue by alanbsmith** [Workday/canvas-kit#1311]
## 💡 Suggestion

Bump `uuid` dependency. The versions we're using now, `^3.3.3` and `^3.4.4` are deprecated.

## Motivation

Just the usual housekeeping. When we bump uuid from v3 to v4, we’ll need to [update these imports](https://github.com/Workday/canvas-kit/search?q=uuid%2Fv4) to look like this:

```ts
// old
import uuid from 'uuid/v4';

// new
import { v4 as uuidv4 } from 'uuid';
```

**Comment by NicholasBoll** [comment]
I actually plan on getting rid of `uuid` altogether! But we can bump it.

**Comment by NicholasBoll** [comment]
Closed by #1408
